meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot/0001-iMX8M-support-SPL-ddr-sign.patch

```diff
@@ -0,0 +1,41 @@
+From e69abd6123a78a9939e4c9a1c437e89a6a2f5975 Mon Sep 17 00:00:00 2001
+From: Jorge Ramirez-Ortiz <jorge@foundries.io>
+Date: Mon, 2 Nov 2020 13:30:17 +0100
+Subject: [PATCH 1/2] iMX8M: support SPL-ddr sign
+
+When there is no second boot loader, there is no need to fill the CSF
+region with zeroes. Only extend the image to the start of CSF.
+
+The image should be generated as:
+ $ mkimage_imx8 -version v1
+                -fit
+                -loader u-boot-spl-ddr.bin 0x7E1000
+		-out flash.bin
+
```

Comment:
Can you add 'Upstream-Status: Pending' here? Just to easy tracking the OE-patches.

If LMP specific, change from Pending to 'Inappropriate [lmp specific]'.

Reply:
Same goes for all the other patches.

Reply:
@ldts is upstreaming of this patch in progress? If yes, could you please provide the link